<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-DKVFQQ5" failures="1" tests="3" name="Smoke Test" time="9.144" errors="0" timestamp="2023-04-12T19:44:58 IST">
  <testcase classname="com.selenium.testng.tests.LoginTest" name="testInvalidLoginApplication" time="0.119"/>
  <testcase classname="com.selenium.testng.tests.LoginTest" name="testLogOutApplication" time="0.011"/>
  <testcase classname="com.selenium.testng.tests.LoginTest" name="testLoginApllication" time="8.256">
    <failure type="java.lang.AssertionError" message="expected [LogOut] but found [Logout]">
      <![CDATA[java.lang.AssertionError: expected [LogOut] but found [Logout]
at com.selenium.testng.tests.LoginTest.testLoginApllication(LoginTest.java:12)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- testLoginApllication -->
</testsuite> <!-- Smoke Test -->
